sycl/doc/extensions/C-CXX-StandardLibrary/C-CXX-StandardLibrary.rst

```diff
@@ -232,6 +240,30 @@ glibc) has its own wrapper library object:
   - libsycl-glibc.o
   - libsycl-msvc.o
 
+Header-based wrappers
```

Comment:
Hi, @asavonic .
If compiler developer provides some functions in device library via headers, the compiler users should not try to provide their own version in source code. This "user override" behavior is permitted now as all functions in current device library is "weak". Suggest to tell users that they should not try to override any function provided in header-based device library.